addons/pharma/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -413,4 +469,438 @@ class ACE_Medical_Treatment_Actions {
         callbackSuccess = QFUNC(treatmentAdvanced_CheckCoag);
         litter[] = {};
     };
+    class syringe_EACA_5ml_1: Carbonate {
+        displayName = CSTRING(push_EACA_IV_1);
+        displayNameProgress = CSTRING(pushing_EACA_IV_1);
+        allowedSelections[] = {"Body", "LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 1;
+        medicRequired = QGVAR(medLvl_EACA);
+        treatmentTime = QGVAR(treatmentTime_EACA);
+        items[] = {"kat_syringe_eaca_5ml_1"};
+        condition = QUOTE(FUNC(ivCondition));
+        callbackSuccess = QFUNC(medication);
+        sounds[] = {};
+    };
+    class syringe_EACA_5ml_3: syringe_EACA_5ml_1 {
+        displayName = CSTRING(push_EACA_IV_3);
+        displayNameProgress = CSTRING(pushing_EACA_IV_3);
+        items[] = {"kat_syringe_eaca_5ml_3"};
+        callbackSuccess = QFUNC(medication);
```

Comment:
you inherit this value from "parent" so why do you repeat it

Reply:
because when i don't, it breaks, as seeing how everything else in kam has a callbackSuccess = QFUNC(medication); entry, I decided to not touch it

Reply:
What do you mean it breaks

Reply:
It doesnt work, the medication injects and does nothing